operating_unit/README.md

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Please follow the structure of README.rst as here:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Only use relevant chapters. E.g. Known issues / Roadmap is not relevant in this case.